plugins.audio.pbsfm/resources/language/English/strings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
`.xml` language files are deprecated. You should use `.po` files instead: http://kodi.wiki/view/language_support